source/includes/fact-connect-srv-description.rst

```diff
@@ -0,0 +1,28 @@
+*New in Compass version 1.12*
+     
+This toggle indicates whether the provided :guilabel:`Hostname`
+is an
+:manual:`SRV Record </reference/connection-string/#dns-seedlist-connection-format>`.
+
+SRV connection strings have a prefix of ``mongodb+srv:``.
+If you are using an SRV connection string, you do not need to
+include ``mongodb+srv`` in your :guilabel:`Hostname`.
+
+.. example::
+
+    If this is the SRV connection string for your MongoDB instance:
+
+    .. code-block:: shell
+        :copyable: false
+
+        mongodb+srv://<username>:<password>@gettingstarted-7q2cs.mongodb.net/test
+
+    In |compass-short|, you would specify your
```

Comment:
I think here it would make sense to add "Then" to the beginning of the sentence so that the example / thought flows better. (if/then).